persistence/sql/persister_test.go

```diff
@@ -42,21 +43,35 @@ func TestManagers(t *testing.T) {
 		t.Run("package=consent/janitor="+k, testhelpers.JanitorTests(m.Config(), m.ConsentManager(), m.ClientManager(), m.OAuth2Storage()))
 
 		t.Run("package=jwk/manager="+k, func(t *testing.T) {
-			testGenerators := new(driver.RegistryBase).KeyGenerators()
-			for algo, testGenerator := range testGenerators {
-				t.Run("TestManagerKey", func(t *testing.T) {
-					ks, err := testGenerator.Generate("TestManagerKey", "sig")
-					require.NoError(t, err)
-
-					jwk.TestHelperManagerKey(m.KeyManager(), algo, ks, uuid.New())
```

Comment:
These tests were not actually executed and I had to change them because how Run subtest function was called. This led me to change the test to exclude HS256/HS512 key generators because TestHelperManagerKey, TestManagerKeySet were implemented for symmetric keys. Additionally I had to skip TestHelperManagerKey, TestManagerKeySet when HSM enabled because they test adding key/keysets and therefore are not aplicable for HSM.